### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by ghost** [common-voice/common-voice#2533]
It would be great to have a dark theme for the website to spare my eyes in the evening.

**Comment by mnv936** [comment]
I would like work on this feature.

**Comment by limapedro** [comment]
Yeah this would really cool, I used an extension but it was terrible. This one feature is must.

**Comment by shakti97** [comment]
I would like to work on this feature, adding this to my todo list

**Comment by mohamedfenjiro** [comment]
Hey there :)
I have created a PR for this, let me know what do you think as it may need iteration for the colors.
[https://github.com/mozilla/voice-web/pull/2675](url)

**Comment by dag7dev** [comment]
+1

Anyway, you could try CV Project on Android (even if it is unofficial) --> it features dark mode (and other features).

**Comment by NWJ9PB** [comment]
Bumping this issue again as it is _necessary_ in some cases
